pkg/blockstorage/awsefs/awsefs.go

```diff
@@ -372,12 +372,20 @@ func (e *efs) SnapshotCreate(ctx context.Context, volume blockstorage.Volume, ta
 	if err = e.setBackupTags(ctx, *resp.RecoveryPointArn, infraTags); err != nil {
 		return nil, errors.Wrap(err, "Failed to set backup tags")
 	}
+
+	req2 := &backup.DescribeRecoveryPointInput{}
```

Comment:
@PrasadG193 - I think it makes sense to follow up with a PR that removes the code duplication b/w this and `SnapshotGet` - will avoid other instances where we might construct `blockstorage.Snapshot` differently.

Reply:
Not a blocker - should do this in a follow up